db/import_column_family_job.cc

```diff
@@ -55,17 +54,18 @@ Status ImportColumnFamilyJob::Prepare(uint64_t next_file_number,
         }
       }
 
-      std::sort(sorted_files.begin(), sorted_files.end(),
-                [&ucmp](const IngestedFileInfo* info1,
-                        const IngestedFileInfo* info2) {
-                  return sstableKeyCompare(ucmp, info1->smallest_internal_key,
-                                           info2->smallest_internal_key) < 0;
-                });
+      std::sort(
```

Comment:
If I understand correctly, the fix is to avoid the range tombstone check in sstableKeyCompare, which check the case if the two smallest_internal_key is equal it will sort range tombstone sentinel keys before other keys with the same user-key. This compare is not needed when just sorting SST files in this level based on the smallest_internal_key?

Reply:
Right, the special logic in `sstableKeyCompare()` is not relevant here. For checking that files within a level are non-overlapping, we want a simple internal key comparison.

thanks for reviewing!